multi/stm32l4-multi/stm32l4-multi.c

```diff
@@ -87,6 +87,10 @@ static void handleMsg(msg_t *msg)
 			err = flash_writeData(imsg->flash_addr, msg->i.data, msg->i.size);
 			break;
 
+		case flash_switch:
+			err = flash_switchBanks();
```

Comment:
Maybe some magic value? Should this msg come by accident (for any reason), it would be bad.

Reply:
Good point, I applied the magic